rosdep/python.yaml

```
@@ -6611,6 +6611,12 @@ python3-jupyros-pip:
   ubuntu:
     pip:
       packages: [jupyros]
+python3-kaitaistruct:
+  debian: [python3-kaitaistruct]
+  fedora: [python3-kaitaistruct]
+  opensuse: [python3-kaitaistruct]
+  ubuntu:
+    pip: [kaitaistruct]
```

It looks like this is also available on Ubuntu: https://packages.ubuntu.com/search?keywords=python3-kaitaistruct&searchon=names&suite=all&section=all

```suggestion
  ubuntu: [python3-kaitaistruct]
```